src/lib/ChatWindow.vue

```diff
@@ -562,6 +563,13 @@ export default {
     roomInfo() {
       this.$emit('room-info', this.room)
     },
+    onAvatarClick(id) {
+      if (this.room?.isIndividual) {
+        this.roomInfo()
+      } else {
+        this.$emit('room-info', { senderId: id })
```

Comment:
Não podemos usar um nome mais genérico aqui como `userId` ao invés de `senderId`. Me parece que um evento de `room-info` não está atrelado a um sender (ou receiver).

Reply:
Nome do atributo ajustado para `userId`

Reply:
Quanto ao evento `room-info` da linha 570 não estar atrelado a um receiver, esse evento é mandado "para cima", onde o Chat captura no arquivo `index.vue` e faz a chamada da função `openChatInfoSidebar` em `RoomService.js`.

Se não for isso o que você apontou, por favor me fala de novo para eu entender melhor.

Grato pelas sugestões :pray: .